learningresources/models.py

```diff
@@ -61,6 +61,7 @@ class StaticAsset(BaseModel):
     Holds static assets for a course (css, html, javascript, images, etc)
     """
     course = models.ForeignKey(Course)
+    path = models.TextField()  # path in static folder
```

Comment:
I don't think you should need this, this is just the asset.name minus the basepath so you can just replace to get the path.